zipkin-ui/js/publicPath.js

```diff
@@ -0,0 +1,11 @@
+import $ from 'jquery';
+
+// read the public path from the <base> tag where it has to be set anyway because of
+// html-webpack-plugin limitations: https://github.com/jantimon/html-webpack-plugin/issues/119
+// otherwise it could be: window.location.pathname.replace(/(.*)\/zipkin\/.*/, '$1/zipkin/')
+const contextRoot = $('base').attr('href');
```

Comment:
Should we sanity check that it ends with `/`? Looks like the rest of the code assumes that. We could either show an error to the user, or just append `/` here?

Reply:
You're right, I'll append `/` if missing.

Reply:
Done